scripts/updateSources.js

```diff
@@ -216,8 +175,6 @@ async function main() {
     // charts v14
     content = content.replace('getTestView().getCanonicalName();', '"vaadin-charts/" + getTestView().getCanonicalName();');
 
-    content = content.replace('import com.vaadin.flow.demo.ComponentDemoTest','import com.vaadin.tests.ComponentDemoTest');
-    content = content.replace('import com.vaadin.flow.demo.TabbedComponentDemoTest','import com.vaadin.tests.TabbedComponentDemoTest');
```

Comment:
Both classes were coming from the demo-helpers module that has been removed in #2834.